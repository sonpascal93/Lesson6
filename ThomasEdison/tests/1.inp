20
...-
7
V
V
V
V
V
V
V
....-.-.-.
5
ESN
C
ESNC
ESNC
ESNC
-..
7
D
D
D
D
D
D
D
...-....-..--....-...----...-..--.-
9
VSLGSBOB
XK
VSL
GSBOBX
K
VSLGSBO
B
X
K
-.-..----.---.-
4
CJ
K
Q
C
...-.---.....--.....--..--...--..
0
--...-..-
2
G
F
-.--.--.-.----.-......-...-
2
YPYTC
S
---.--.--...--.--..
8
MNTYVTP
E
MNTYVT
P
E
MNTYVTPE
MN
TYVTP
.-..
2
L
L
...-.--.--......--..--...
5
V
W
WHIZGI
VWWHI
Z
-....-...--...-
8
BAV
D
A
BAV
D
A
B
AVDA
...-.--.-..-..-.-....---.-..-
4
VPTUFBW
NX
VPTUFBWN
X
-......-.--
0
--.--.-
2
G
Q
-.-.-.....---.
3
NN
BUTN
NNB
.-.---....-.--.--
0
...----.-.-
1
VOR
.-....-..-..-.-.--.--.-..-..--..
9
LFLCMP
XI
Z
LFLCMP
XIZ
LFLCMPXIZ
LF
LC
MPXI
...--.-..-
7
VNX
V
N
X
VN
X
VN
