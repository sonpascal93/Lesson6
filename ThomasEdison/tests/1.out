7
4
7
8
1
0
0
0
7
2
1
9
2
0
1
1
0
0
6
7
